tests/integration/JobWithFailureErrorLogTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yokai\Batch\Sources\Tests\Integration;
+
+use Doctrine\Persistence\ManagerRegistry;
+use Psr\Log\LoggerInterface;
+use Yokai\Batch\BatchStatus;
+use Yokai\Batch\Job\JobInterface;
+use Yokai\Batch\Job\JobWithChildJobs;
+use Yokai\Batch\JobExecution;
+use Yokai\Batch\Storage\JobExecutionStorageInterface;
+
+class JobWithFailureErrorLogTest extends JobTestCase
```

Comment:
I don't think an integration test is expected for that kind of feature
Maybe you should just add your test case here: https://github.com/yokai-php/batch-src/blob/0.x/src/batch/tests/JobExecutionTest.php